test/README.md

```diff
@@ -28,10 +23,21 @@ OSBUILD_URL="https://localhost:8085" OSBUILD_CA_PATH="/etc/osbuild-composer/ca-c
 OSBUILD_CERT_PATH="/etc/osbuild-composer/client-crt.pem" OSBUILD_KEY_PATH="/etc/osbuild-composer/client-key.pem"
 ```
 
+Otherwise set some bogus values:
+```
+COMPOSER_TOKEN_URL=bogus COMPOSER_CLIENT_ID=bogus COMPOSER_CLIENT_SECRET=bogus
+```
+
+Start Image Builder:
+```
+PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=foobar PGDATABASE=imagebuilder
+DISTRIBUTIONS_DIR="/$HOME/image-builder/distributions" ./image-builder
```

Comment:
Is it possible to set the bogus value in a command separate from the one to start Image Builder?

Reply:
Well  it's implied that you should add those when invoking image-builder. So You'd need to just add them, or export them.

Reply:
I'm asking because I probably wouldn't get that if I haven't tried it before. Maybe `Start Image Builder with the paths or bogus values` could help to be more explicit? Not entirely sure so just adding that as a thought.

Reply:
But otherwise looks good to me :)

Reply:
hm, it's just a bash thing tho, idk